docs/developer-guide/build-zone-setup-triton.md

```diff
@@ -178,8 +178,15 @@ So, the two ways of creating build zones are:
  * Choose a CN to provision to (selecting "headnode" will ensure the build zone
    is provisioned on that headnode)
  * Choose the "external" network
- * Add the following Customer metadata, `{"user-script": "/usr/sbin/mdata-get root_authorized_keys > ~root/.ssh/authorized_keys ; /usr/sbin/mdata-get root_authorized_keys > ~admin/.ssh/authorized_keys; svcadm enable manifest-import"}`
+ * Add the following Customer metadata, `{"user-script": "/usr/sbin/mdata-get root_authorized_keys > ~root/.ssh/authorized_keys ; /usr/sbin/mdata-get root_authorized_keys > ~admin/.ssh/authorized_keys; svcadm enable manifest-import"}` along with supplying your public ssh key.
  * Click "Provision machine"
+ * If you encounter any problems while sshing to your build zone, zlogin into it and verify that the `manifest-import` service is online by doing this:
+```
+svcs -p manifest-import
+STATE          STIME    FMRI
+online         20:42:28 svc:/system/manifest-import:default
+```
+
```

Comment:
This probably isn't the best for brand new users, since they won't have any context on how to access the compute-zone in order to zlogin into their zone. I'd be more inclined to leave out that section and perhaps use something along the lines of:

_
* Click "Provision machine"

During the build zone's first-boot as part of provisioning, the `manifest-import` service can take a few minutes to complete. Since the `manifest-import` service is responsible for importing and enabling the `ssh` service, you will have to wait for that to complete before being able to login to the zone.
_